docs/FAQ.md

```diff
@@ -0,0 +1,23 @@
+# FAQ
+
+**What does OA stand for?**
+
+ocm-agent, also OAO stands for ocm-agent-operator.
+
+**What is ocm-agent?**
+
+ocm-agent is a web-service managed by OAO as a funnel point for all managed clusters related to any OCM interaction. 
+
+**What are ocm-agent use-cases?**
+
+1. Provide a service endpoint for the publishing of service logs to the customer from other on-cluster workloads.
+
+> **_NOTE_** OCM Agent use-cases will grow as the project adds abilietis.
+
+**Does OA silence alerts?**
+
+No. Alert on-boarded to OCM Agent will not reach Pager-Duty but will have a service-log sent automatically.
+
+**How does OA determine which service log template to use?**	
+
+OAO manages managed-notifications API and in turn managednotifications CRD, in association with the CR when an alert is on-boarded will have a template config stored and will be watched by ocm-agent to send the service log.
```

Comment:
1. It should be better to use Camel case for the resource, `managedNotifications CR`
2. `config stored` vs `config to store`?

Reply:
Sounds good, I will make this change.

Reply:
Done.